examples/zenohpico/z_pull.c

```diff
@@ -0,0 +1,100 @@
+//
```

Comment:
These are Zenoh-Pico examples.
Why do we have them in Zenoh-CPP as well?

Reply:
see above